go.mod

```diff
@@ -258,7 +260,6 @@ replace (
 
 	github.com/golang/protobuf => github.com/golang/protobuf v1.4.2
 	github.com/grpc-ecosystem/grpc-gateway => github.com/grpc-ecosystem/grpc-gateway v1.16.0
-	github.com/improbable-eng/grpc-web => github.com/improbable-eng/grpc-web v0.0.0-20181111100011-16092bd1d58a
```

Comment:
I'm assuming this is no longer required in grpc-web v.0.15.0

Reply:
Correct me if I am wrong. I think since Go 1.17, all these `replace` lines are not necessary, all indirect dependencies are listed as required and their version can be bumped from there

> At `go 1.17` and above, the go command adds an indirect requirement for each module that provides any package imported (even [indirectly](https://go.dev/ref/mod#glos-indirect-dependency)) by a package or test in the main module or passed as an argument to `go get`.

https://go.dev/ref/mod#go-mod-file-require

So it should be possible to clean up most of what's in `replace` here

Reply:
Possibly... I wish a comment had been added to justify the replace. Some have been added to resolve CVEs and stuff like that. I'll leave the cleanup for another PR.